src/PKSim.Infrastructure/Services/ObservedDataTask.cs

```diff
@@ -62,6 +66,13 @@ public void ExportToPkml(DataRepository observedData)
          _observedDataPersistor.Save(observedData, file);
       }
 
+      public async Task<IReadOnlyList<DataRepository>> LoadFromSnapshot()
+      {
+         var observedData = (await _snapshotTask.LoadFromSnapshot<DataRepository>()).ToList();
+         observedData.Each(AddObservedDataToProject);
+         return observedData.ToList();
```

Comment:
why ToList here? this is already a list
What does LoadFromSnapshot returns

Reply:
actually you probably do not need the returned value here as this is typically called from a UICommand? Simply return Task no?